src/transactions/test/PathPaymentTests.cpp

```diff
@@ -112,6 +112,78 @@ TEST_CASE("pathpayment", "[tx][pathpayment]")
 
     closeLedgerOn(*app, 2, 1, 1, 2016);
 
+    SECTION("transact more than INT64_MAX in a path payment")
+    {
+        auto a1 = root.create("A1", minBalance4);
+        auto a2 = root.create("A2", minBalance4);
+        auto a3 = root.create("A3", minBalance4);
+
+        a1.changeTrust(idr, trustLineLimit);
+        a1.changeTrust(cur2, trustLineLimit);
+
+        a2.changeTrust(cur2, trustLineLimit);
+        a2.changeTrust(cur1, trustLineLimit);
+
+        a3.changeTrust(idr, trustLineLimit);
+        a3.changeTrust(cur2, trustLineLimit);
+        a3.changeTrust(cur1, trustLineLimit);
+
+        SECTION("issue more than INT64_MAX")
```

Comment:
I'm not seeing how this shows that more than `INT64_MAX` was issued. Unless I'm missing something, it looks to me like you have demonstrated issuing `INT64_MAX` and transferring `INT64_MAX/2`. This is interesting in its own right, but it's not what the test claims.